.github/workflows/publish.yaml

```diff
@@ -1,6 +1,7 @@
 name: PyPI-publish
 
 on:
+  workflow_dispatch:
```

Comment:
Out of curiosity, what is this for?

Reply:
this creates the manually-run workflow button!